rules/autoconsent/zentralruf-de.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "zentralruf-de",
+  "prehideSelectors": ["#cookie_modal_wrapper"],
+  "detectCmp": [{ "exists": "#cookie_modal_wrapper" }],
```

Comment:
@seia-soto I think this detection selector may cause [false positives](https://publicwww.com/websites/%22cookie_modal_wrapper%22/). Looking at the code, the pop-up looks site-specific, should we add a "urlPattern"?

Reply:
Sorry, I should be more careful. Thanks for the review.

Reply:
Updated.